RELEASE.md

```diff
@@ -49,7 +49,8 @@ It may also be useful for release management purposes to ensure that no unexpect
 It is released to [GitHub releases](https://github.com/deephaven/deephaven-core/releases) in the [syft](https://github.com/anchore/syft) json format.
 
 ### Deephaven python client
-The Deephaven python client is released as the `pydeephaven` wheel at [PyPi](https://pypi.org/project/pydeephaven/).
+The Deephaven python client is released as the `pydeephaven` wheel at [PyPi](https://pypi.org/project/pydeephaven/) and
+`pydeephaven-ticking` wheels at [PyPi](https://pypi.org/project/pydeephaven-ticking/).
```

Comment:
Seems strangely worded, where the link is on PyPi both times. Suggest switching it around:
```suggestion
The Deephaven python client wheels are released on PyPi as [`pydeephaven`](https://pypi.org/project/pydeephaven/) and
[`pydeephaven-ticking`](https://pypi.org/project/pydeephaven-ticking/).
```